packages/lib/gpc/src/gpcCompile.ts

```diff
@@ -387,6 +394,71 @@ function compileProofObject(objInfo: CompilerObjInfo<POD>): ObjectModuleInputs {
   };
 }
 
+function compileProofBoundsChecks<
+  ObjInput extends POD | GPCRevealedObjectClaims
+>(
+  boundsCheckConfig: GPCProofBoundsCheckConfig,
+  entryMap: Map<PODEntryIdentifier, CompilerEntryInfo<ObjInput>>,
+  paramBoundsChecks: number
+): {
+  boundsCheckEntryIndices: CircuitSignal[];
+  boundsCheckMinValues: CircuitSignal[];
+  boundsCheckMaxValues: CircuitSignal[];
+} {
+  // Arrange POD entry identifiers alphabetically
```

Comment:
This is more subtle than it seems at first.  POD names aren't cryptographically enforced, so they're arbitrary.  We already depend on their order being consistent, but not on the names being unchanged.  Sorting by a PODEntryIdentifier as a string is going to depend on more than just the name order, but on their length too, given how lexicographic sorting works.

We do need a consistent order, but I think it should directly mirror object and entry order.  Concretely, I think that means you need to implement a comparison function which sorts strictly by objectName then EntryName, treated as a tuple rather than a single string.